### DISCUSSION TITLE
Does fluent-bit opentelemetry input support https?

### DISCUSSION THREAD

**Discussion by fmhwong** [fluent/fluent-bit#9104] [General]
I have an application that sends logs using OTLP to an opentelemetry collector.  The collector will forward the logs to fluent-bit using otlp-http.  Fluent-bit is configured with opentelemetry input on port 4318.  I'd like to secure the communications with https.  Does fluent-bit opentelemetry input support https?

**Comment by fmhwong** [comment]
Looks like I found the answer [here](https://docs.fluentbit.io/manual/administration/transport-security).

**Reply by agup006** [reply]
Yes, it does. Similar to network based inputs you can use the docs linked. Thanks for positing those here @fmhwong
